source/common_fields.rst

```diff
@@ -0,0 +1,484 @@
+
+.. image:: ../image/test-rule-location.png
+   :alt: module assets - add rule location
+   :align: center
+   :scale: 67%
+
+Manufacturer
+~~~~~~~~~~~~~
+
+Manufacturers are automatically retrieved by the automatic inventory and assigned to the computer. you can also create/add them manually.
+
+.. image:: ../image/manufacturer.png
+   :alt: module assets - computer-manufacturer
+   :align: center
+   :scale: 50%
+
+- To create a manufacturer, click **+**
+- Add a **Name**
+- You can add a registrered ID issued by `PCI-SIG <https://pcisig.com/developers/integrators-list>`_ for USB or/and PCI. You can add more by clicking **+**.
+
+Once the manufacturer has been created, you can add another or select it from the drop-down list.
+
+.. image:: ../image/add-manufacturer.png
+   :alt: module assets - add computer manufacturer
+   :align: center
+   :scale: 63%
+
+
+Model
+~~~~~
+
+When the inventory is uploaded, the template field is filled in automatically. You can also create/add templates manually.
+Modeles can also be used by templates and rules.
+
+.. image:: ../image/model.png
+   :alt: module assets - computer model
+   :align: center
+   :scale: 50%
+
+- To create a model, click **+**
+- Add a **Name**
+- Add a **Product number** if it's necessary
+- You can specify the **Weight** and the **Depth**, the **Required units** (in the case of a rack server)
+- You can indicate the **Power connections** and the **Power consumption**
+- Indicate whether it is a half rack
+- Add pictures (front, rear, other)
+
+.. image:: ../image/add-model-1.png
+   :alt: module assets - computer model
+   :align: center
+   :scale: 63%
+
+.. image:: ../image/add-model-2.png
+   :alt: module assets - computer model
+   :align: center
+   :scale: 63%
+
+
+Example with picture :
+
+.. image:: ../image/rack-model.png
+   :alt: module assets - rack model
+   :align: center
+   :scale: 42%
+
+If you insert a machine with photos in a rack, the photos will be visible in that rack
+
+Example of a server in a rack with photo :
+
+.. image:: ../image/example-rack-model.png
+   :alt: module assets - rack view
+   :align: center
+   :scale: 67%
+
+Network
+~~~~~~~
+
+**Network** is information added manually. you can pre-fill this information using a template.
+
+.. image:: ../image/network.png
+   :alt: module assets - network
+   :align: center
+   :scale: 50%
+
+
+- To create a network, click **+**
+- Enter a **Name**
+- Enter a **Comment** (optionnal)
+- Click **+ Add**
+- You can add another one or close this window
+- Then select the network you want from the drop-down list
+
+.. image:: ../image/add-network.png
+   :alt: module assets - add network
+   :align: center
+   :scale: 63%
+
+
+Serial number
+~~~~~~~~~~~~~
+
+This information is automatically retrieved by the automatic inventory. You can add/modify this information manually. By default, if you add or modify this field

```

Comment:
Globally, all fields can be locked. I guess it would be better to have a lock § somewhere, rather than explain lock on each related field.

Reply:
Right, i add an admonition tip with a link to lock page